.github/workflows/docker.yml

```
@@ -0,0 +1,23 @@
+name: Docker
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Build and Run
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Run the Docker image
+        run: |
+          mkdir kb
+          DOCKER_BUILDKIT=1 docker build . -t ubuntu/sc-machine:local
+          docker run --rm -e REBUILD_KB=1 -v ${ls}/kb:/kb -v kb_binary:/kb.bin ubuntu/sc-machine:local serve -t -h 0.0.0.0 -c sc-machine/config/sc-machine.ini
```

Sorry, my bad, forgot about `/`
```suggestion
          docker run --rm -e REBUILD_KB=1 -v ${ls}/kb:/kb -v kb_binary:/kb.bin ubuntu/sc-machine:local serve -t -h 0.0.0.0 -c /sc-machine/config/sc-machine.ini
```